src/fastqc/config.vsh.yaml

```
@@ -0,0 +1,182 @@
+name: fastqc
+namespace: fastqc
```

No need for a namespace if there is just one command

```suggestion
name: fastqc